src/entities/swap/swaps/v2/auraBalSwaps/replaceWrapped.ts

```diff
@@ -0,0 +1,13 @@
+import { NATIVE_ASSETS, ZERO_ADDRESS } from '@/utils';
+import { Address } from 'viem';
+
+export function replaceWrapped(tokens: Address[], chainId: number): Address[] {
```

Comment:
Unfortunately the other one uses `Token` array instead of `Address`. I did think about altering it to handle both but I think its better to keep it more specific.